sampleconfig/orderer.yaml

```diff
@@ -68,14 +68,14 @@ General:
         # ServerPrivateKey defines the file location of the private key of the TLS certificate.
         ServerPrivateKey:
 
-    # Genesis method: The method by which the genesis block for the orderer
+    # Bootstrap method: The method by which to obtain the bootstrap block
     # system channel is specified. The option can be one of:
```

Comment:
this comment doesn't read correctly anymore...

Reply:
Sorry, but what's wrong here?